docs/quickstarts/twitter.rst

```diff
@@ -54,8 +54,51 @@ run:
     $ export OAUTHLIB_INSECURE_TRANSPORT=1
     $ python twitter.py
 
-Visit http://localhost:5000 in your browser, and you should start the OAuth dance
-immediately.
+Visit ``http://127.0.0.1:5000`` in your browser, and you should start the OAuth dance
+immediately after you click on the Signin with Twitter hyperlink.
+
+In your view, make sure you use a plain hyperlink to begin the dance
+so that the redirect loads Twitter's confirmation page in the browser.
+
+.. code-block:: HTML
+
+    <a href="/twitter/login">Sign in with Twitter</a>
+
+You can only use http libraries like axios to check the
+authentication status.
+
+For instance, when the component mounts, you can call a function to check
+the authentication status. If the authentication fails, then
+you can display a hyperlink for the user to begin the dance, or
+if it succeeds, then simply display the authenticated username.
+
+The following example uses axios and vuejs but this could be ported to
+react, angular, or vanila javascript. You could create 3 state variables:
+welcome (String ""), authenticated (Boolean false),
+authenticateCheckComplete (Boolean false), and then use these to
+either show the hyperlink or the authenticated username.
```

Comment:
Why are we talking about Javascript libraries in the Twitter quickstart? This doesn't belong here at all.

Reply:
As you recommended, I'll move this into a new doc directed at React/Angular/VueJS devs and then we can link to the new document. That's a great compromise - thanks!